CMakeLists.txt

```diff
@@ -123,15 +123,16 @@ endif (NOT APPLE AND NOT MINGW AND NOT MSVC)
 option(AEOLUS        "Enable pipe organ synthesizer"      OFF)
 option(ZERBERUS      "Enable experimental SFZ sampler"    ON)
 option(OSC           "Enable OSC remote control protocol" ON)
-option(OMR           "Enable PDF import"  OFF)                 # OMR - optical music recognition
+option(AVSOMR        "Enable AVSOMR import"               ON)               # AVSOMR - optical music recognition by audirevis
+option(OMR           "Enable PDF import"                  OFF)                 # OMR - optical music recognition
 # For installation see: http://ubuntuforums.org/showthread.php?t=1647350
-option(OCR           "Enable OCR, requires OMR" OFF)           # Requires tesseract 3.0, needs work on mac/win
-option(SOUNDFONT3    "Ogg Vorbis compressed fonts" ON)         # Enable Ogg Vorbis compressed fonts, requires Ogg & Vorbis
-option(HAS_AUDIOFILE "Enable audio export" ON)                 # Requires libsndfile
+option(OCR           "Enable OCR, requires OMR"           OFF)           # Requires tesseract 3.0, needs work on mac/win
+option(SOUNDFONT3    "Ogg Vorbis compressed fonts"        ON)         # Enable Ogg Vorbis compressed fonts, requires Ogg & Vorbis
+option(HAS_AUDIOFILE "Enable audio export"                ON)                 # Requires libsndfile
```

Comment:
can you lign the comments too please?

Reply:
Yes, of course I did. Sorry, I would be in a hurry :)